examples/app-recreate.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
What about calling the file `app-setup.sh` and having also `app-teardown.sh` with `flask db destroy --yes-i-know`?